legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/DDL/toDDL.pure

```diff
@@ -101,13 +101,13 @@ function meta::relational::functions::database::loadCsvDataToDbTable(csvData:Lis
                                 | let schemaName = $tab.p.values->at(0).values->at(0);
                                   let tableName = $tab.p.values->at(1).values->at(0);
                                   let columnNames = $tab.p.values->at(2).values;
-                                  let schema = $db->schema($schemaName);
-                                  assertNotEmpty($schema, | 'Schema "' + $schemaName + '" not found in database "' + $db->elementToPath() + '"');
+                                  let schema = $db->map(d|$d->schema($schemaName))->toOne();
+                                  assertNotEmpty($schema, | 'Schema "' + $schemaName + '" not found in database "' + $db->at(0)->elementToPath() + '"');
                                   let table = $schema->toOne()->table($tableName);
-                                  assertNotEmpty($table,| 'Table "' + $tableName + '" not found in Schema "' + $schemaName + '" in Database "' + $db->elementToPath() + '"');
+                                  assertNotEmpty($table,| 'Table "' + $tableName + '" not found in Schema "' + $schemaName + '" in Database "' + $db->at(0)->elementToPath() + '"');
                                   let pTable = $tablePostProcessor->eval($table)->toOne();
                                   let columns = $columnNames->map(cName | let c = $pTable->toOne().columns->cast(@Column)->filter(tc| $tc.name->equalIgnoreCase($cName->trim()))->first();
-                                                                          assertNotEmpty($c,| 'Column "' + $cName + '" not found in Table "' + $tableName + '" in Database "' + $db->elementToPath() + '"');
+                                                                          assertNotEmpty($c,| 'Column "' + $cName + '" not found in Table "' + $tableName + '" in Database "' + $db->at(0)->elementToPath() + '"');
                                                                           $c;  );
```

Comment:
same comment on the above, we should list all dbs if schema, table or coolumn not found.